operator/model/src/test/java/io/apicurio/registry/operator/api/v3/SerDeTest.java

```diff
@@ -0,0 +1,25 @@
+package io.apicurio.registry.operator.api.v3;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SerDeTest {
+
+    static final ObjectMapper MAPPER = new ObjectMapper(new YAMLFactory());
+
+    @Test
+    public void basicReadOfCRs() throws Exception {
+        // Arrange
+        var resource = getClass().getResourceAsStream("/demo-cr.yaml");
+
+        // Act
+        var registry = MAPPER.readValue(resource, ApicurioRegistry3.class);
+
+        // Assert
+        assertEquals("test", registry.getMetadata().getName());
+        assertEquals("test-namespace", registry.getMetadata().getNamespace());
```

Comment:
I'd like to see a comparison with a manually (with code) created instance (with status, since spec is empty). Builders help here.